.github/workflows/e2e.yaml

```diff
@@ -0,0 +1,120 @@
+name: e2e tests
+on:
+  pull_request:
```

Comment:
Probably we would like to run it also on schedule as for elemental we used to have it. To be sure that `master` branch is tested regulary

```
on:
  schedule:
    - cron: 0 22 * * *
```